python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -155,10 +155,13 @@ def __init__(self):
         }
 
         self.q = PriorityQueue()
+        self.disconnected = False 
```

Comment:
why disconnected is `False` at init time? I would imagine it is disconnected at this time, no?